mesonbuild/utils/universal.py

```diff
@@ -1154,10 +1154,12 @@ def join_args(args: T.Iterable[str]) -> str:
 
 
 def do_replacement(regex: T.Pattern[str], line: str,
-                   variable_format: Literal['meson', 'cmake', 'cmake@'],
+                   variable_format: Literal['meson', 'cmake', 'cmake@', 'autoconf'],
```

Comment:
Perhaps creating a helper type instead of just repeating this over and over.

Reply:
Done